crates/types/src/config/rocksdb.rs

```diff
@@ -98,6 +98,16 @@ pub struct RocksDbOptions {
     /// Default: 64MB
     #[cfg_attr(feature = "schemars", schemars(with = "Option<NonZeroByteCount>"))]
     rocksdb_log_max_file_size: Option<NonZeroByteCount>,
+
+    /// # RocksDB block size
+    ///
+    /// Uncompressed block size
+    ///
+    /// Default: 64KB
```

Comment:
Are we increasing the default to 64KB to increase the effect of compression and reduce the number of entries in the index block of a table file?

Reply:
Yes. larger blocks is more EBS friendly (less IOPS) and allow for better compression, and since we are now compressing L0 files, we'll always be writing much smaller blocks than what's configured here.

Reply:
Thanks for the clarification. Makes sense :-)